gobblin-data-management/src/main/java/org/apache/gobblin/data/management/copy/recovery/RecoveryHelper.java

```diff
@@ -122,8 +120,8 @@ public boolean persistFile(State state, CopyableFile file, Path path) throws IOE
       return Optional.absent();
     }
 
-    Path glob = new Path(this.persistDir.get(), computeGuid(state, file) + "_*");
-    for (FileStatus fileStatus : this.fs.globStatus(glob)) {
+    Path guidPath = new Path(this.persistDir.get(), computeGuid(state, file));
+    for (FileStatus fileStatus : this.fs.listStatus(guidPath)) {
```

Comment:
This may throw a `FileNotFoundException`.

Reply:
Added try/catch for `FileNotFoundException`